src/System.Private.CoreLib/shared/System/Text/EncodingData.cs

```diff
@@ -0,0 +1,298 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// THIS IS AUTOGENERATED FILE
```

Comment:
@tarekgh Is it correct? Where does the source code for the tool live?

Reply:
I think we should change the license to match all our sources. I'll try to point where the tool code today as I am catching up with  the issues.

Reply:
Here is where we stored the tool: 

https://github.com/dotnet/buildtools/blob/6736870b84e06b75e7df32bb84d442db1b2afa10/src/Microsoft.DotNet.Build.Tasks/PackageFiles/encoding.targets
https://github.com/dotnet/buildtools/blob/6736870b84e06b75e7df32bb84d442db1b2afa10/src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs#L16

In early stage, we included the tool as an msbuild target in buildtools. we can move this to be stand alone as needed.

Reply:
https://github.com/dotnet/corefx/pull/2718